internal/cmd/versions_cmd_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd_test
```

Comment:
notice the file name.. I'm just dodging a name collision as we can't have white box (same package) and black box (package_test) in the same file.. I'm happy to rename this now or some later time when we find a better practice. I don't think there's a known better way in go... https://stackoverflow.com/questions/19998250/proper-package-naming-for-testing-with-the-go-language/31443271#31443271

Reply:
thanks for clarifying. Let's come to this later 😄  i think for now it's ok to leave it as-is